# practise problems
# a = input("sir please enter your name")
# print(a,"good afternoon")
# # or
# print("good afternoon, "+a)
# # or you can style the above code as follows
# b = input("sir please enter the input\n")
# print(a,"hello rajesh")
# letter = '''Dear ,<!NAME!>,
#                   you are selected!
#                   <!DATE!>'''
# a = letter.replace("<!NAME!>","rajesh")
# b = a.replace("<!DATE!>","28/03/2021")
# print(b)
# print("or")

# letter = letter.replace("<!NAME!>","rajesh")
# letter= letter.replace("<!DATE!>","28/03/2021")
# print(letter)
# you can use any of the following to do the things you need
# wrong method
# a = letter.replace("<!NAME!>","rajesh")
# a = letter.replace("<!DATE!>","28/03/2021")
# print(a)
# the above method doesn't work the above one is completely a rejected method do not repeat the mistake
# print("or")
# you can write the above code as and many things are same but you see them different
# ab = input("enter the name: ")
# ac = input("enter the date: ")
# letter = letter.replace("<!NAME!>","rajesh")
# letter= letter.replace("<!DATE!>","28/03/2021")
# print(letter)
# strings = "hello  this is rajesh"
# print(strings.find("  "))
# p = strings.replace("  "," ")
# print(p)
# hellos = "shiva is one of the great ,gods"
# hello = "shiva is one of the great \,gods"
# print(hellos)
# print(hello)
# hello guys now we are going to do formating
# helloshiva = "Dear Harry,This Python Course is nice.Thanks!"
# helloshiva = "Dear Harry,\nThis Python Course is nice.\nThanks!"
# print(helloshiva)
letter = "shiva"
print(letter[0])
letter[0] = "P"
print(letter[0])